packages/cli/src/commands/site/deploy/index.ts

```diff
@@ -98,6 +99,12 @@ const command: CommandModule<
           );
         }
 
+        if (args.version != null && !isValidSemver(args.version)) {
+          throw new YargsCheckError(
+            "--version must be a valid SemVer string",
```

Comment:
```suggestion
            `Version "${args.version}" must be a valid SemVer string",
```

Reply:
I feel leaving the `--version` could be helpful to clearly show this is coming directly from the user passing `--version` and not possibly due to autoVersion failing in some form?

```suggestion
          `--version "${args.version}" must be a valid SemVer string`,
```

Reply:
Looks like you made most of these existing error messages reference the option name so fine to keep it like that!